plugins/hydra_optuna_sweeper/setup.py

```diff
@@ -32,7 +32,7 @@ def get_long_description() -> str:
         "Development Status :: 4 - Beta",
     ],
     install_requires=[
-        "hydra-core",
+        "hydra-core>1.0",
```

Comment:
This expresses the intention better in my opinion:
```
hydra-core>=1.1
```

Reply:
Thank you for your suggestion, but I got the following installation error with `hydra-core>=1.1` when I ran the nox's `"lint_plugins-3.8` task.

```console
ERROR: Could not find a version that satisfies the requirement hydra-core>=1.1 (from hydra-optuna-sweeper)              ERROR: No matching distribution found for hydra-core>=1.1                                                               
WARNING: You are using pip version 20.3.1; however, version 21.0.1 is available.                                        
You should consider upgrading via the '/home/yanase/pfn/code/hydra/.nox/lint_plugins-3-8/bin/python -m pip install --upgrade pip' command.                                                                                                      
nox > Session lint_plugins-3.8 failed. 
```

I think this is because `hydra-core==1.1.0` has not been released yet, and I reverted the change with TODO comment (https://github.com/facebookresearch/hydra/pull/1294/commits/0fe11b699ce60e0622e4570a1b331caa3e74e500).